packages/eslint-plugin-svelte/src/rules/no-inspect.ts

```diff
@@ -0,0 +1,30 @@
+import type { TSESTree } from '@typescript-eslint/types';
+
+import { createRule } from '../utils';
+
+export default createRule('no-inspect', {
+	meta: {
+		docs: {
+			description: '',
+			category: 'Best Practices',
+			recommended: true,
```

Comment:
Could you set it to `false` since it would be a breaking change? Since we want to change it in a major version, could you leave a comment `// TODO Switch to recommended in the major version.` ?